Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/AbilData.xml

```diff
@@ -14751,4 +14765,73 @@
         </InfoArray>
         <Flags index="WarpTrainersButton" value="0"/>
     </CAbilWarpTrain>
+    <CAbilTransport id="AP_InfBullfrogTransport">
+        <EditorCategories value="Race:Zerg,AbilityorEffectType:Units"/>
+        <AbilSetId value="ULdM"/>
+        <CmdButtonArray index="Load" DefaultButtonFace="AP_InfBullfrogLoad"/>
+        <CmdButtonArray index="LoadAll">
+            <Flags index="Hidden" value="1"/>
+        </CmdButtonArray>
+        <CmdButtonArray index="UnloadAll">
+            <Flags index="Hidden" value="1"/>
+            <Flags index="ToSelection" value="1"/>
+        </CmdButtonArray>
+        <CmdButtonArray index="UnloadAt" DefaultButtonFace="AP_InfBullfrogUnloadAll"/>
+        <CmdButtonArray index="UnloadUnit">
+            <Flags index="Hidden" value="1"/>
+        </CmdButtonArray>
+        <MaxCargoCount value="8"/>
+        <MaxCargoSize value="4"/>
+        <TotalCargoSpace value="8"/>
+        <UnloadPeriod value="1"/>
+        <LoadCargoBehavior value="TransportDummy"/>
+    </CAbilTransport>
+    <CAbilEffectTarget id="AP_InfBullfrogLeapUnload">
+        <Alignment value="Negative"/>
+        <EditorCategories value="Race:Zerg,AbilityorEffectType:Units"/>
+        <Effect index="0" value="AP_BullfrogLeapUnloadCreatePlaceholder"/>
+        <Flags index="RequireTargetVision" value="0"/>
+        <Cost>
+            <Cooldown TimeUse="24"/>
+        </Cost>
+        <Range value="14"/>
+        <UninterruptibleArray index="Prep" value="1"/>
+        <UninterruptibleArray index="Cast" value="1"/>
+        <UninterruptibleArray index="Channel" value="1"/>
+        <UninterruptibleArray index="Finish" value="1"/>
+        <PrepTime value="0.5"/>
+        <CmdButtonArray index="Execute" DefaultButtonFace="AP_RavagerCorrosiveBile"/>

```

Comment:
Add CursorEffect to AP_InfBullfrogLandingSearch

That'd allow target highlighting